oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +344,40 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+
+        int nameStart = stat.indexOf("(");
+        int nameEnd = stat.indexOf(")");
+        if (Util.isBlank(status.get("Name")) && nameStart < nameEnd) {
+            // remove leading and trailing parentheses
+            String statName = stat.substring(nameStart, nameEnd + 1);
```

Comment:
shouldn't this be `nameStart + 1` to exclude the leading parenthesis and `nameEnd` (which is exclusive) to exclude the ending parenthesis?

Reply:
I did output the original name from the status map and the name I extract here, the indexes are right as they are (on Ubuntu 20.04 LTS with Java 1.8_265)

Reply:
Ah, I see what you're doing. In this line you're capturing the full string with both parentheses, but then in the next line you're starting the substring from 1, and going to length-1 (which, since it's exclusive, has length-2 as last character), effectively removing the two parenthesis.  If you just use `start+1, end` on this line, you don't need a second line.

Reply:
Oh yeah, thats still in because I had the string split at first, obviously can be put together. I just woke up so I could do it in half an hour.